src/app/settings/WishListSettings.tsx

```diff
@@ -74,12 +77,19 @@ export default function WishListSettings() {
     dispatch(clearWishLists());
   };
 
-  const changeUrl = (url: string, enabled: boolean) => {
+  const changeUrl = (url: string, enabled: boolean, manualRefresh?: boolean | false) => {
     const toAddOrRemove = validateWishListURLs(url);
     const newUrls = enabled
       ? [...activeWishlistUrls, ...toAddOrRemove.filter((url) => !activeWishlistUrls.includes(url))]
       : [...activeWishlistUrls.filter((url) => !toAddOrRemove.includes(url))];
-    reloadWishList(newUrls.join('|'));
+    reloadWishList(newUrls.join('|'), manualRefresh);
+  };
+
+  const reloadAllWishListsEvent = () => {
+    // for each active URL, fire a reload
+    for (const activeWishlistUrl of activeWishlistUrls) {
+      changeUrl(activeWishlistUrl, true, true);
+    }
```

Comment:
I don't think you need to fire a refresh for each URL, since `reloadWishList` already takes all the URLs. You probably want this to be:
```suggestion
    reloadWishList(activeWishlistUrls.join('|'), true);
```

Reply:
I actually originally did this, but saw that changeUrl was a wrapper and bet against myself. Also, I mistakenly thought it wasn't working because of the GitHub caching and so your change I agree makes more sense now.